.gitignore

```diff
@@ -7,14 +7,20 @@ target/
 build/
 buildNest/
 venv/
-spinnaker-install/
+tests/spinnaker_tests/bak/spinnaker-install/
```

Comment:
Could you remove this file from the PR and also all of the SpiNNaker-2 related files from this PR? Cheers!